Makefile Windows

```diff
@@ -6,7 +6,7 @@ URCHIN_DIR=./cmd/urchin
 URCHIN_ADMIN_DIR=./cmd/urchin-admin
 
 # Name of the binary
-BINARY_NAME=urchin
+BINARY_NAME=urchin.exe
```

Comment:
I still don't think we need this `Makefile Windows` file - the windows `.air_windows.toml` file doesn't reference it at all.